### PR TITLE
Update wrap regform section in jinja blocks

### PR DESCRIPTION
**Pull request by SegiNyn**

### CODE REVIEW

File: indico/modules/events/registration/templates/management/regform.html [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+        {% elif regform.is_open %}
+            {% include 'events/registration/management/regform_actions/open.html' %}
+        {% elif regform.has_ended %}
+            {% include 'events/registration/management/regform_actions/ended.html' %}
+        {% endif %}
+    {% endblock %}
 
-    <div class="action-box">
-        {% block regform_setup %}
+    {% block regform_setup %}
```
Comment (OmeGak) [inline indico/modules/events/registration/templates/management/regform.html:14]: To give a bit more context, I would consider naming this block `regform_setup_box`:

```suggestion
    {% block regform_setup_box %}
```

File: indico/modules/events/registration/templates/management/regform_list.html [hunk 2]
```diff
@@ -13,9 +13,9 @@
 
 {% block content %}
     {% if not event.has_feature('payment') %}
-        <div id="payment-disabled-notice" class="action-box highlight">
-            <div class="section">
-                {% block payments_disabled %}
+        {% block payments_disabled %}
```
Comment (OmeGak) [inline indico/modules/events/registration/templates/management/regform_list.html:16]: Again, I would consider adding `_box` to the block name: 

```suggestion
        {% block payments_disabled_box %}
```

File: indico/modules/events/registration/templates/management/regform_list.html [hunk 3]
```diff
@@ -34,13 +34,13 @@
...
-                {% endblock %}
+                </div>
             </div>
-        </div>
+        {% endblock %}
     {% endif %}
 
-    <div class="action-box">
-        {% block global_registration_actions %}
+    {% block global_registration_actions %}
```
Comment (OmeGak) [inline indico/modules/events/registration/templates/management/regform_list.html:42]:
```suggestion
    {% block global_registration_actions_box %}
```

File: indico/modules/events/management/templates/_management_frame.html [hunk 4]
```diff
@@ -24,7 +24,7 @@
 {%- endmacro -%}
 
 
-{% macro render_event_management_header_right(event, event_types) %}
+{% macro render_event_management_header_right(event, event_types, with_clone_info=true, with_action_menu=true) %}
```
Comment (ThiefMaster) [inline indico/modules/events/management/templates/_management_frame.html:27]: Maybe add a comment right above it so the args aren't accidentally removed at some point?

```suggestion
{# The extra args are not used in Indico but are there on purpose for others overriding the template. #}
{% macro render_event_management_header_right(event, event_types, with_clone_info=true, with_action_menu=true) %}
```

**Pull request merged**